joint_trajectory_controller/doc/userdoc.rst

```diff
@@ -257,10 +356,16 @@ The following version of the Joint Trajectory Controller are available mapping t
 ..   - velocity_controllers::JointTrajectoryController
 ..     - Input: position, [velocity, [acceleration]]
 ..     - Output: velocity
-.. TODO(anyone): would it be possible to output velocty and acceleration?
+.. TODO(anyone): would it be possible to output velocity and acceleration?
```

Comment:
This should go because it is not possible in the current implemenation

Reply:
I removed it. How does the namespacing work as described in this section? I did not understand this up to now..